SocialCareCaseViewerApi.Tests/V1/IntegrationTests/IntegrationTestHelpers.cs

```diff
@@ -113,6 +113,9 @@ public static CreateAllocationRequest CreateAllocationRequest(long personId, int
                 .RuleFor(c => c.MosaicId, f => personId)
```

Comment:
would be good to start building integration tests for all different allocation use cases to build confidence in the new implementation